auth/README.md

```diff
@@ -397,16 +397,47 @@ represented in the following diagram:
 
 ## UI customization
 
-To provide customization of the visual style of the activities that implement
-the flow, a new theme can be declared. Standard material design color
-and typography properties will take effect as expected. For example, to define
-a green theme:
+To use FirebaseUI Auth's sign-in flows, you must provide an `app_name` string and use the
+AppCompat color attributes in your app.
+
+For example, something like this goes in your `strings.xml`:
+```xml
+<resources>
+    <!-- ... -->
+    <string name="app_name">My App</string>
+</resources>
+```
+
+This goes in your `styles.xml`:
+```xml
+<style name="AppTheme" parent="FirebaseUI">
+    <!-- Required for sign-in flow styling -->
+    <item name="colorPrimary">@color/material_green_500</item>
+    <item name="colorPrimaryDark">@color/material_green_700</item>
+    <item name="colorAccent">@color/material_purple_a700</item>
+</style>
+```
+You don't have to call `setStyle()` on the sign in intent builder as FirebaseUI
+will automatically style the activities. For example:
+
+```java
+startActivityForResult(
+    AuthUI.getInstance(this).createSignInIntentBuilder()
+        .build());
+```
+
+Note: If your app is already using an AppCompat theme, you don't have to create 
+a new theme, but you can still optionally style other parts of the sign-in flow. 
+Standard material design colorand typography properties will take effect as expected. 
+For example, to define a green theme:
```

Comment:
I think this paragraph up to `You don't have to call...` should be replaced with something simpler since we don't talk about `setTheme` anywhere else:
```
If you would like more control over FirebaseUI's styling, you can define your own custom style to
override certain or all styling attributes. For example, a green sign-in theme:
```

PS: we don't talk about `setTheme` anywhere except [here](https://github.com/firebase/FirebaseUI-Android/pull/862/files/8b8707c244580002d809342f715ac89d129e05e4#diff-46b10a690795758a8db6b0f86c9afddcR216) which we should remove.